stubs/D3DShot/d3dshot/display.pyi

```diff
@@ -0,0 +1,49 @@
+from ctypes import _Pointer
+from typing_extensions import TypedDict
+
+from d3dshot.dll import _ProcessFunc, _ProcessFuncRegionArg, _ProcessFuncReturn
+from d3dshot.dll.d3d import ID3D11Device, ID3D11DeviceContext
+from d3dshot.dll.dxgi import IDXGIAdapter, IDXGIOutput1, IDXGIOutputDuplication
+
+class _PositionDict(TypedDict):
+    left: int
+    top: int
+    right: int
+    bottom: int
+
+# Note that Display.d3d_device and Display.d3d_device_context can never be None.
+# Despite initially being set to None in __init__,
+# they're always immediatly set in _initialize_dxgi_output_duplication()
```

Comment:
Extremely picky nitpick: this comment is quite a long way from the attributes it's commenting on; I'd prefer it if it was immediately above those attributes. Also, spelling -- should be "immediately", not "immediatly" :)